Documentation/ceph-osd-mgmt.md

```diff
@@ -66,6 +66,18 @@ and it is safe to proceed. If an OSD is failing, the PGs will not be perfectly c
 Update your CephCluster CR. Depending on your CR settings, you may need to remove the device from the list or update the device filter.
 If you are using `useAllDevices: true`, no change to the CR is necessary.
 
+> **IMPORTANT:** On host-based clusters, you may need to stop the Rook Operator while performing OSD
```

Comment:
Instead of "IMPORTANT", how about a titled section such as:
```
#### Clean the Disk to Avoid OSD Re-Creation

Reply:
I like "IMPORTANT" so readers know to pay special attention to the content. If they don't do this and the disk is not totally failed, Rook will re-create the OSD and mess up user attempts to replace it. I don't see that a new section is really necessary or helpful.

Reply:
What do you think about the rendered version with it just all bold @travisn ?